picard/ui/itemviews.py

```diff
@@ -256,16 +263,16 @@ def update_selection(self, i, j):
         self.window.update_selection(
             [item.obj for item in self.views[i].selectedItems()])
 
-    def update_selection_0(self):
+    def update_selection_clusters(self):
         if not self._ignore_selection_changes:
             self._ignore_selection_changes = True
-            self.update_selection(0, 1)
+            self.update_selection(Panels.CLUSTERS.value, Panels.ALBUMS.value)
             self._ignore_selection_changes = False
 
-    def update_selection_1(self):
+    def update_selection_albums(self):
         if not self._ignore_selection_changes:
             self._ignore_selection_changes = True
-            self.update_selection(1, 0)
+            self.update_selection(Panels.ALBUMS.value, Panels.CLUSTERS.value)
```

Comment:
Finally I'm not very happy with Panels enum, we have a lot of code redundancy in there.
I suggested few changes in https://github.com/Gabrielcarvfer/picard/pull/1